From 7ad32737be019230b9219f0b46916c569edad941 Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Tue, 14 Jul 2020 09:09:29 +0700
Subject: [PATCH 022/433] ARM: dts: r8a7743: Add power-domain for CPU0 and CPU1
 of r8a7743

This patch adds power-domain managed by SYSC for
CPU0 and CPU1 of R8A7743.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/r8a7743.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index 8c8bb62..68851d1 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -48,9 +48,11 @@
 			compatible = "arm,cortex-a15";
 			reg = <0>;
 			clock-frequency = <1500000000>;
+			voltage-tolerance = <1>; /* 1% */
 			clocks = <&cpg_clocks R8A7743_CLK_Z>;
 			clock-latency = <300000>; /* 300 us */
 			next-level-cache = <&L2_CA15>;
+			power-domains = <&sysc R8A7743_PD_CA15_CPU0>;
 
 			/* kHz - uV - OPPs unknown yet */
 			operating-points = <1500000 1000000>,
@@ -69,6 +71,7 @@
 			clocks = <&cpg_clocks R8A7743_CLK_Z>;
 			clock-latency = <300000>; /* 300 us */
 			next-level-cache = <&L2_CA15>;
+			power-domains = <&sysc R8A7743_PD_CA15_CPU1>;
 
 			/* kHz - uV - OPPs unknown yet */
 			operating-points = <1500000 1000000>,
-- 
2.7.4

